src/services/company.js

```diff
@@ -175,6 +176,26 @@ class CompanyService {
         await this._sendCompanyNotification(companyId, COMPANY_ENABLED_NOTIFICATION);
     }
 
+    async sendCompanyDeletedNotification(companyId) {
+        await this._sendCompanyNotification(companyId, COMPANY_DELETED_NOTIFICATION);
+    }
+
+    /**
+     * Deletes a company by its ID and returns it
+     * @param {*} companyId ID of the company
+     */
+    deleteCompany(companyId) {
```

Comment:
I think this should be changed to deleteCompanyAndAccount for clarity, as it should delete both

Reply:
I deleted the account in the endpoint to then use it to send the email